### PR TITLE
Retrieved root folder info for the bucket

### LINKED ISSUE #23: Add GCS bucket root folder into the output
We currently don't include GCS bucket root folder in the report which makes it hard to follow and identify sensitive buckets. We need to include them too.

### PR DESCRIPTION
**Pull request by aswin2108**
## Changes
Added lines to retrieve the root folder information for each bucket and then root folder information is then added to the `buckets_dict`.
Fix #23 

## Checklist
 - [x] I have read and followed the contributing guidelines.
 - [ ] I have tested my changes thoroughly and they work as expected.
 - [ ] I have added the necessary tests for the changes made.
 - [ ] I have updated the documentation to reflect the changes made.
 - [x] My code follows the project's coding style and standards.
 - [x] I have added appropriate commit messages and comments for my changes. 

## Additional Note
The code (before the changes) was not running properly on my end, trying to figure it out, hence I couldn't test it out. So can someone pls help me through it? Thought the code I added would be useful in some way! Did I do it right? What all further changes must be made?

### CODE REVIEW

**Comment by mshudrak** [comment]
I'd suggest you to setup a test GCP project and check why the changes does not work by debugging and printing all the necessary variables in console. So, far I see this
```
googleapiclient.errors.HttpError: <HttpError 404 when requesting https://storage.googleapis.com/storage/v1/b/gcf-sources-413204024550-us-central1/o/gcf-sources-413204024550-us-central1?alt=json returned "No such object: gcf-sources-413204024550-us-central1/gcf-sources-413204024550-us-central1". Details: "[{'message': 'No such object: gcf-sources-413204024550-us-central1/gcf-sources-413204024550-us-central1', 'domain': 'global', 'reason': 'notFound'}]">
```

**Comment by Bhardwaj-Himanshu** [comment]
Hi @aswin2108, 
Coming here from #23, and wanted to know the updates about the issue!
Also if I could be of any help, do let me know.

**Comment by mshudrak** [comment]
No response on this one. Archiving.

**Comment by aswin2108** [comment]
Sorry for the late reply! Got held up! Actually, I couldn't resolve it, so @Bhardwaj-Himanshu if you want take it!

**Comment by Bhardwaj-Himanshu** [comment]
Hi @mshudrak, I know this issue has been closed! But if this dosen't cause any trouble to you, can we reopen this issue?
Also can you tell me the importance of this bug--like how is it crashing or obstructing the software?

and Thanks @aswin2108, you are always welcomed to help me mate!
